migrations/5_remove_holding_accounts.coffee

```diff
@@ -12,14 +12,14 @@ module.exports = HoldingAccountMigration =
 
 	deleteUserProjects: (user_id, callback = (error) ->) ->
 		# Holding accounts can't own projects, so only remove from 
-		# collaberator_refs and readOnly_refs
+		# collaborator_refs and readOnly_refs
 		console.log "[Removing user from projects]", user_id
 		db.projects.find {
 			$or: [
-				{collaberator_refs: user_id},
+				{collaborator_refs: user_id},
```

Comment:
This typo is unfortunate, but all the data in the production DB (and DBs of Server CE/Pro users) uses this field already. Fixing it requires a lot more work (changes in web + a migration + documentation + major release for Server CE/Pro). I do not think it is worth the effort at this time -- cc @gh2k @jdleesmiller @mserranom @ShaneKilkelly .
https://github.com/overleaf/web/blob/2ca61ecc075a855df8a45df58b1cb67ff3b20f73/app/src/models/Project.js#L47

Reply:
Yeah, we have some of these in the schemas and we can't realistically fix them.